themes/default/content/blog/pulumi-security-posture/index.md

```diff
@@ -0,0 +1,97 @@
+---
+title: "Pulumi Security Posture"
+
+date: 2022-10-06
+
+meta_desc: Pulumi Security Posture.  Organization Access Token, Team Access Token, Teams, and Managed Federated Identity
+
+meta_image: meta.png
+
+authors:
+    - tushar-shah
+
+tags:
+    - security
+    - pulumi-enterprise
+
+---
+
+Cloud computing’s greatest strength and weakness is the proliferation of a massive number of services globally. In order to adequately assess and mitigate the inherent risks for your company, your customers, and your employees, Cloud Architects are typically responsible for a massive surface area of potential endpoints and vectors of attack.
+
+<!--more-->
+
+## Cloud Security Posture
+
+AWS likes to talk about a [Shared Responsibility Model](https://aws.amazon.com/compliance/shared-responsibility-model/), their friendly way of saying, “we’ve got our stuff covered and if you’re compromised, it’s almost certainly your fault.”  They have built up practices and recommendations like the AWS Well Architected Framework; pillars of architecture philosophy that give you your best chances to securely use their resources (following these recommendations is easy with [Crosswalk for AWS](https://www.pulumi.com/docs/guides/crosswalk/aws/) from Pulumi).  Increasing high profile attacks against private and open source software supply chains has developers increasingly on high alert and conscious of incorporating more Sec into their DevOps.
```

Comment:
@tusharshahrs Would you mind going through and replacing all of your `https://www.pulumi.com/` links to use Hugo `relref` shortcodes? Like this (and be careful to remove any trailing slashes as you do so)? Thank you! 

```suggestion
AWS likes to talk about a [Shared Responsibility Model](https://aws.amazon.com/compliance/shared-responsibility-model/), their friendly way of saying, “we’ve got our stuff covered and if you’re compromised, it’s almost certainly your fault.”  They have built up practices and recommendations like the AWS Well Architected Framework; pillars of architecture philosophy that give you your best chances to securely use their resources (following these recommendations is easy with [Crosswalk for AWS]({{< relref "/docs/guides/crosswalk/aws" >}}) from Pulumi).  Increasing high profile attacks against private and open source software supply chains has developers increasingly on high alert and conscious of incorporating more Sec into their DevOps.
```

Reply:
Updated the urls